README.md

```diff
@@ -1 +1,119 @@
-# my-portforlio-project
\ No newline at end of file
+📗 Table of Contents
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+🚀 Live Demo
+💻 Getting Started
+Setup
+Prerequisites
+Install
+Usage
+Run tests
+Deployment
+👥 Authors
+🔭 Future Features
+🤝 Contributing
+⭐️ Show your support
+🙏 Acknowledgements
+❓ FAQ (OPTIONAL)
+📝 License
+📖 [your_project_name]
+My portfolio project
+
+[your_project__name] is a portfolio project built with HTML and CSS
+
+🛠 Built With
+Tech Stack
+HTML and CSS
+
+Client
+Server
+Database
+Key Features
+
+[Linters]: This is a cool tool that helps avoid coding error
+[package.json]: Holds all the dependencies of our project
+[stylelintrc]: Helps avoid errors in css file
+[HTML]: is a template used to structure our website
+[CSS]: For styling
+
+🚀 Live Demo
+Add a link to your deployed project.
+
+💻 Getting Started
+This are the tools ylu need to get started
+* A working internet
+* linters
+* A vs code editor
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+In order to run this project you need:
+
+Setup
+Clone this repository to your desired folder:
+
+Install
+Install this project with:
+
+Usage
+To run the project, execute the following command:
+
+Run tests
+To run tests, run the following command:
+
+Deployment
+You can deploy this project using:
+
+(back to top)
+
+👥 Authors
+Kingsley Chidozie Igbor
+* Github: @Developer-Kingz
+* LinkedIn: https://www.linkedin.com/in/kingsley-igbor-14b04720b/
+* Twitter: @DaKingzMan
```

Comment:
As requested by the previous reviewer, currently you have only included the link to your `LinkedIn` profile.

Please kindly include the link to the remaining of your social media profiles for a professional readme. 👍
i.e. 
- Github 
- Twitter